cirq/contrib/svg/svg.py

```diff
@@ -6,12 +6,25 @@
     import cirq
 
 QBLUE = '#1967d2'
+FONT = "Arial"
+
+
+def fixup_text(text: str):
+    if '\n' in text:
+        return '?'
+    if '[<virtual>]' in text:
```

Comment:
This seems awfully specific.

Reply:
We can wait for https://github.com/quantumlib/Cirq/issues/2984#issuecomment-638427120 and then it will be unnecessary but I'm using this patch currently

Reply:
This is contrib, so I'd just make sure we have an issue to fix this before moving to cirq, and maybe a link from that issue.

Reply:
there's a comment linking to the issue; I'll put a backref on the issue itself